charts/incubator/ghostfolio/templates/common.yaml

```diff
@@ -0,0 +1,53 @@
+{{/* Make sure all variables are set properly */}}
+{{- include "tc.v1.common.loader.init" . }}
+
+{{/* Render configmap for ghostfolio */}}
+{{- $configmapPaths := include "ghostfolio.configmap.paths" . | fromYaml -}}
+{{- if $configmapPaths -}}
+  {{- $_ := set .Values.configmap "ghostfolio-paths" $configmapPaths -}}
+{{- end -}}
+
+{{- $configmapFile := include "ghostfolio.configmap.configfile" . | fromYaml -}}
+{{- if $configmapFile -}}
+  {{- $_ := set .Values.configmap "ghostfolio-configfile" $configmapFile -}}
+{{- end -}}
+
+{{/* Render secrets for ghostfolio */}}
+{{- $secret := include "ghostfolio.secrets" . | fromYaml -}}
+{{- if $secret -}}
+  {{- $_ := set .Values.secret "ghostfolio-secrets" $secret -}}
+{{- end -}}
+
+{{/* Append the general configMap volume to the volumes */}}
+{{- define "ghostfolio.configmapVolume" -}}
+enabled: true
+mountPath: /.env
+subPath: .env
+readOnly: true
+type: "configmap"
+objectName: ghostfolio-configfile
+{{- end -}}
+
+{{/* Append the general secret volumes to the volumes */}}
+{{- define "ghostfolio.secretVolumes" -}}
+enabled: true
+mountPath: "/secrets"
+readOnly: true
+type: "secret"
+objectName: ghostfolio-secrets
+items:
+  - key: "ACCESS_TOKEN_SALT"
+    path: ACCESS_TOKEN_SALT
+  - key: "JWT_SECRET_KEY"
+    path: JWT_SECRET_KEY
+  - key: "DATABASE_URL"
+    path: DATABASE_URL
+  - key: "REDIS_PASSWORD"
+    path: REDIS_PASSWORD
+{{- end -}}
```

Comment:
There is no need for those to be in the common.yaml.
There is no conditionals. Please move them to values.yaml